json_schema_fuzz/schema_operations.py

```diff
@@ -313,11 +328,17 @@ def invert(
         })
 
     additional_properties = schema.get("additionalProperties", None)
-    if additional_properties:
+    if additional_properties is not None:
         inverted_schemas.append({
             "anyAdditionalProperties": invert(additional_properties),
         })
 
+    any_additional_property = schema.get("anyAdditionalProperty", None)
```

Comment:
If we aren't adding something like `minAdditionalProperties` right now, I think it might be good to have a different name that implies "at least one must exist". How about "someAdditionalProperty" as in "some additional property must satisfy..." whereas "additionalProperties" means "all additional properties must satisfy..."?